.github/workflows/integrational-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Integrational tests
+on:
+  push:
+    paths-ignore:
+      - 'doc/**'
+
+jobs:
+  run-tests:
+    name: Run Intergrational tests using Kind
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.7.0"
+
+      - name: Deploy to Kuberenetes
+        run: |-
+          kubectl apply -f rbac.yaml
+          kubectl apply -f config.yaml
+          kubectl apply -f deployment.yaml
+
+      - name: Test for successful Deployment
+        run: |-
+          kubectl wait --for=condition=ready pod/hazelcast-sidecar-0 --timeout=150s
+          kubectl wait --for=condition=ready pod/hazelcast-sidecar-1 --timeout=120s
```

Comment:
Is that even possible to have these two pods as separate clusters? I mean, once `hazelcast-sidecar-1` is ready, is it certain that they formed a cluster with `hazelcast-sidecar-0`?

Reply:
Well, I think it is. We can parse the logs to be sure or try to write the value to `hazelcast-sidecar-0` and read it from `hazelcast-sidecar-1` which will also test that the application is connected to hazelcast.
Will update the test

Reply:
I reconsidered it and started feeling over-suspicious about that. Because what I said
> Is that even possible to have these two pods as separate clusters?

actually attempts to test the kubernetes discovery here (or validate it on a dockerized k8s env.). May be we should skip this, I'm not sure. WDYT @leszko?

Reply:
@enozcan I have added the test for the python application, so in this case, besides testing the cluster creation it also tests that the application does connect to the Hazelcast sidecar container